src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -26,14 +26,14 @@
 from dmg_utils import get_dmg_command
 from fault_config_utils import FaultInjection
 from general_utils import \
-    get_partition_hosts, stop_processes, \
-    get_default_config_file, pcmd, get_file_listing, DaosTestError, run_command, \
-    dump_engines_stacks
+    get_partition_hosts, stop_processes, get_default_config_file, pcmd, get_file_listing, \
+    DaosTestError, run_command, dump_engines_stacks, get_avocado_config_value, \
+    set_avocado_config_value
 from logger_utils import TestLogger
 from pydaos.raw import DaosContext, DaosLog, DaosApiError
 from server_utils import DaosServerManager
 from test_utils_container import TestContainer
-from test_utils_pool import TestPool, LabelGenerator
+from test_utils_pool import LabelGenerator, add_pool, POOL_NAMESPACE
```

Comment:
(pylint-syntax-error) Cannot import 'test_utils_pool' due to syntax error 'invalid syntax (<unknown>, line 53)'